mythril/laser/ethereum/state/environment.py

```diff
@@ -41,7 +41,10 @@ def __init__(
         self.active_function_name = ""
 
         self.address = active_account.address
+
+        # TODO: Block number should increase with transactions
```

Comment:
This is not necessarily true. A series of transactions can occur within one block. 
Although there would be a constraint that (new blocknumber >= previous blocknumber)